README.md

```diff
@@ -169,6 +169,7 @@ SIG-specific GitHub discussions.
 | Functions as a Service (FAAS)         | Every other Tuesday at 12:00 PM PT                                                                         | [Google Doc](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc/edit#)                       | [#otel-faas](https://cloud-native.slack.com/archives/C04HVBETC9Z)                                                                                            |
 | Profiling WG                          | Every other week on Thursday at 08:00 AM PT                                                                | [Google Doc](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit?usp=sharing)            | [#otel-profiles](https://cloud-native.slack.com/archives/C03J794L0BV)                                                                                        |
 | Security                              | Weekly on Wednesday at 08:30 AM PT                                                                         | [Google Doc](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg/edit?usp=sharing)            | [#otel-sig-security](https://cloud-native.slack.com/archives/C05A85QC281)                                                                                    |
+| OpenTelemetry on Mainframes           | Weekly on Tuesday at 10:00 AM PT                                                                         | [Google Doc](https://docs.google.com/document/d/14p-bpofozTL4n3jy6HZH_TKjoOXvog18G1HBRqq6liE/edit?usp=sharing)            | [#otel-mainframes](https://cloud-native.slack.com/archives/C05PXDFTCPJ)                                                                                      |
```

Comment:
@dyladan can you transfer the google doc to the GC account?

Reply:
yes